### DISCUSSION TITLE
Add grid filtering

### DISCUSSION THREAD

**Discussion by wolph** [djangopackages/djangopackages#666] [Grids]
It would be very useful to me if the grid supported a little bit of filtering.

For example, if a package was last updated in 2015 (there are several packages like that) it is most likely not functioning anymore so I'm not immediately interested in that package anymore.

I'm willing to implement it, but it depends a bit on how it should be implemented. I think a few lines of javascript would already offer a quick version of this. Alternatively it could be done in Python but that would be slower and will increase server load as well.

**Comment by AntonOfTheWoods** [comment]
Did this get moved forward at all? It would be much more useful to be able to filter on Python3 compatibility or last updated...
